draft-ietf-taps-interface.md

```diff
@@ -737,23 +737,39 @@ early binding when required, for example with some Network Address Translator
 
 ### Using Multicast Endpoints
 
-Specifying a multicast group address on a Local Endpoint will indicate to the Transport
-Services system that the resulting connection will be used to receive multicast messages. The
-Remote Endpoint can be used to filter incoming multicast from specific senders. Such
-a Preconnection will only support calling Listen(), not Initiate(). Calling Listen()
-will cause the Transport Services system to register for receiving multicast, such
-as issuing an IGMP join {{?RFC3376}} or using MLD for IPV6 {{?RFC4604}}. Any Connections
-that are accepted from this Listener are receive-only.
+To use multicast, a Preconnection is first created with the Remote Endpoint
+specifying the multicast group and destination port number.
+
+Calling Initiate() on that Preconnection creates a Connection that can be
+used to send messages to the multicast group. The Connection object that is
+created will support Send() but not Receive(). Any Connections created this
+way are send-only, and do not join the multicast group. The resulting
+Connection will have a Local Endpoint indicating the local interface to
+which the connection is bound and a Remote Endpoint indicating the
+multicast group.
+
+Calling Listen() on a Preconnection with a multicast group as the Remote
```

Comment:
I am unsure on how you would join SSM with these semantics. Maybe add an example for that as well? Where do you specify the source address?

Reply:
As discussed in the interim, we will habe an extra attribute for the remote endpoint to specify the multicast group.
```suggestion
Calling Listen() on a Preconnection with a multicast group specified on the Remote
```